README.md

```diff
@@ -59,9 +59,11 @@ Run `--help` on any `blast` command to show all available subcommands, parameter
 
 ```bash
 blast --help
+blast node --help
+blast node start --help
```

Comment:
will this show any different `help` information than the `blast node --help` or the ` or onyl the `blast help`

Reply:
`blast node --help` will show help for all available subcommands and `blast node start --help` will show the same description for `start` and also any optional parameters for `start` with usage